inc/core/patterns/bcgov-alpha3-front-page.php

```diff
@@ -0,0 +1,98 @@
+	<!-- /wp:paragraph --></div>
+	<!-- /wp:column -->
+	
+	<!-- wp:column -->
+	<div class="wp-block-column"></div>
+	<!-- /wp:column --></div>
+	<!-- /wp:columns --></div>
+	<!-- /wp:group --></div></div>
+	<!-- /wp:cover -->
+	
+	<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50","left":"2rem","right":"2rem"}}},"layout":{"type":"constrained","contentSize":"1440px"},"metadata":{"name":"Quick Access"}} -->
+	<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--50);padding-right:2rem;padding-bottom:var(--wp--preset--spacing--50);padding-left:2rem"><!-- wp:group {"style":{"spacing":{"margin":{"bottom":"var:preset|spacing|40"}}},"layout":{"type":"default"},"metadata":{"name":"BC Gov H2"}} -->
+	<div class="wp-block-group" style="margin-bottom:var(--wp--preset--spacing--40)"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+	<div class="wp-block-group"><!-- wp:group {"style":{"border":{"bottom":{"color":"#f8ba47","width":"6px"},"top":{},"right":{},"left":{}}},"className":"is-heading-two-border","layout":{"type":"constrained","justifyContent":"left","contentSize":"40px"}} -->
+	<div class="wp-block-group is-heading-two-border" style="border-bottom-color:#f8ba47;border-bottom-width:6px"></div>
+	<!-- /wp:group --></div>
+	<!-- /wp:group -->
+	
+	<!-- wp:heading {"style":{"elements":{"link":{"color":{"text":"var:preset|color|black"}}}},"textColor":"black","fontSize":"large"} -->
+	<h2 class="wp-block-heading has-black-color has-text-color has-link-color has-large-font-size">Call-to-Action Section</h2>
+	<!-- /wp:heading --></div>
+	<!-- /wp:group -->
+	
+	<!-- wp:columns {"className":"is-style-constrained is-triple-card-layout"} -->
+	<div class="wp-block-columns is-style-constrained is-triple-card-layout"><!-- wp:column {"className":"is-style-default"} -->
+	<div class="wp-block-column is-style-default"><!-- wp:group {"style":{"spacing":{"padding":{"right":"0rem","left":"0rem"}}},"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group" style="padding-right:0rem;padding-left:0rem"><!-- wp:image {"id":1377,"aspectRatio":"16/9","scale":"cover","sizeSlug":"full","linkDestination":"none","style":{"border":{"radius":"5px"}}} -->
+	<figure class="wp-block-image size-full has-custom-border"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/default_01.jpg" alt="Pear" class="wp-image-1377" style="border-radius:5px;aspect-ratio:16/9;object-fit:cover" title=""/></figure>
+	<!-- /wp:image -->
+	
+	<!-- wp:heading {"style":{"color":{"text":"#1a5a96"},"elements":{"link":{"color":{"text":"#1a5a96"}}},"typography":{"fontStyle":"normal","fontWeight":"700"}},"className":"has-no-underline","fontSize":"small"} -->
```

Comment:
This RGB value looks to be close to the default --wp--preset--color--secondary-brand color. Should this have a WordPress colour class applied rather than an explicit color value? Eg: class="has-secondary-brand-color"

Reply:
That's a good point. Personally I feel like these values should be static because we're following a design system and the colours should remain consistent with that. If you disagree then I'm open to conversation about it though.

Reply:
It's not a problem as the colours can be edited in the editor. And as it's tied to the concept of the Alpha v3 theming having it set to a specific option is ok. If the Alpha design tokens were more explicitly defined for use, I'd say it would be worth ensuring it used a theme option but for the near term this is cool. It does open a larger question as to how the Block Theme should implement the Design Tokens themselves, but for now it has a reasonable association for their use.